native/cocos/platform/android/adpf_manager.h

```diff
@@ -140,17 +155,29 @@ class ADPFManager {
     jobject obj_power_service_;
     jmethodID get_thermal_headroom_;
 
+    std::map<std::string, jobject> map_hint_sessions;
```

Comment:
It is better to use `std::unordered_map<>` if the order is not important.

Reply:
Thanks, order isn't important, only the key value mapping is, changed in 1d6c8b9359